airflow/dags/copy_prod_archiver_configs_to_test/copy_prod_archiver_configs_to_test.py

```diff
@@ -0,0 +1,55 @@
+from calitp_data_infra.storage import GTFSDownloadConfigExtract, get_latest
+from google.cloud import storage
```

Comment:
Other places we use [get_fs](https://github.com/cal-itp/data-infra/blob/main/packages/calitp-data-infra/calitp_data_infra/storage.py#L49-L55) which I think handles a bit of the local vs. cloud auth logic, and it uses [`gcsfs`](https://gcsfs.readthedocs.io/en/latest/api.html) which has a `copy` function that I think we can just use